lusidtools/cocoon/utilities.py

```diff
@@ -1470,14 +1470,14 @@ def remap_after_merge(mapping: dict, buy_suffix: str, sell_suffix: str) -> dict:
     file_type = "transactions"
     logging.info(f"updating mapping to new Total Consideration and transaction fields ")
     # currencies and amounts coming into the portfolio i.e. buy
-    buy_fields = [
+    sell_fields = [
         "total_consideration.amount",
         "total_consideration.currency",
         "settlement_currency",
     ]
 
     # currencies and amounts leaving the portfolio i.e. sell
-    sell_fields = ["units", "transaction_currency"]
+    buy_fields = ["units", "transaction_currency"]
```

Comment:
How are we certain these are always the `buy` fields?
For example, consider a `FxBuy` of GBP/USD.
Here the units and **transaction_currency** are **indeed** the buy fields.
But what if we're doing a `SellFx` of GBP/USD.
Then the **units** and **transaction_currency** are the sell fields.

Reply:
in  the latter case, wouldn't the units and transaction still need to be mapped the same, but in the configuration they would have opposite sides?

Reply:
So in the standard LUSID canonical transaction types, `SellFx` has the following side configuration:

- CashCommitment on Side1 with a -1 direction
- CashCommitment on Side2 with a 1 direction

In plain English, that means we're:

- Decreasing the cash of the **transaction_currency** by the **units**
- Increasing the cash in the settlement_currency by the **total_consideration.amount**

Therefore labelling **transaction_currency** and **units** as buy_fields is not always correct?

(apologies if I am missing something)